backend/src/services/__tests__/memberService.test.ts

```diff
@@ -138,7 +138,7 @@ describe('MemberService tests', () => {
             default: attributes[MemberAttributeName.LOCATION][PlatformType.GITHUB],
           },
         },
-        email: member1.email,
+        emails: member1.emails,
```

Comment:
Can we add a test-case where we merge 2 members with different emails?

Reply:
updated [this test case](https://github.com/CrowdDotDev/crowd.dev/pull/672/commits/f04612d9ecebda2e196fdf5fa5eef640b6ff19d4)
to include different emails while upserting.
We're already testing the merge function with different emails [here](https://github.com/CrowdDotDev/crowd.dev/blob/f04612d9ecebda2e196fdf5fa5eef640b6ff19d4/backend/src/services/__tests__/memberService.test.ts#L1690)